securedrop/tests/utils/env.py

```diff
@@ -34,16 +34,15 @@ def teardown(config):
     for t in threading.enumerate():
         if t.is_alive() and not isinstance(t, threading._MainThread):
             t.join()
+
+    # Delete the DB file
     db.session.remove()
-    try:
-        shutil.rmtree(config.TEMP_DIR)
-    except OSError:
-        # Then check the directory was already deleted
-        assert not os.path.exists(config.TEMP_DIR)
-    try:
-        shutil.rmtree(config.SECUREDROP_DATA_ROOT)
```

Comment:
I am no longer deleting the `SECUREDROP_DATA_ROOT` between each test because it will delete the `GPG_KEY_DIR` too (as it's a subfolder of `SECUREDROP_DATA_ROOT`).